man/gnt-instance.rst

```diff
@@ -700,9 +700,10 @@ security\_domain
 kvm\_flag
     Valid for the KVM hypervisor.
 
-    If *enabled* the -enable-kvm flag is passed to kvm. If *disabled*
-    -disable-kvm is passed. If unset no flag is passed, and the
-    default running mode for your kvm binary will be used.
+    If *enabled* accel=kvm is appended to the -machine parameter
+    *Disabling* (or not setting this flag at all) currently does
+    nothing. Since KVM is enabled by default anways, this flag is
+    only kept for backwards compability of Ganeti.
```

Comment:
I'm not sure about enabled by default. Maybe this is only the case when the `/usr/bin/kvm` executable is used. Setting `kvm_path` to `qemu-system-x86_64` could result in [tcg](https://manpages.debian.org/jessie/qemu-system-x86/qemu-system-x86_64.1.en.html#OPTIONS).

Reply:
That is true, I'll give that another spin.

At least we can say for sure there is no `-disable-kvm` parameter (and according to my research never has been). But since the code always checked the KVM help output for the existand of said parameter first, it never caused any errors. But also no notifications that disabling KVM did not succeed.

Reply:
I have updated the documentation to reflect the current state.